ansible/roles/riakcs/tasks/admin_user.yml

```diff
@@ -12,7 +12,7 @@
 
 # Note this can't be notify because we need it to restart immediately
 - name: Restart RiakCS
-  command: riak-cs restart
+  service: name=riak-cs state=started enabled=yes
```

Comment:
you're right, this should just be named Start RiakCS. we just need to make sure it's running. the command below needs to be a restart for the changes to take effect. will update that